e2e/cypress/integration/accessibility/accessibility_modals_dialogs_spec.js

```diff
@@ -126,51 +126,60 @@ describe('Verify Accessibility Support in Modals & Dialogs', () => {
-                cy.get('#sidebarPublicChannelsMore').should('have.attr', 'aria-label', 'See more public channels').click();
+                cy.get('#sidebarPublicChannelsMore', {timeout: TIMEOUTS.ONE_MIN}).
+                    should('be.visible').
+                    and('have.attr', 'aria-label', 'See more public channels').click();
 
                 // * Verify the accessibility support in More Channels Dialog`
-                cy.get('#moreChannelsModal').should('have.attr', 'role', 'dialog').and('have.attr', 'aria-labelledby', 'moreChannelsModalLabel').within(() => {
-                    cy.get('#moreChannelsModalLabel').should('be.visible').and('contain', 'More Channels');
-                    cy.get('.modal-header button.close').should('have.attr', 'aria-label', 'Close');
-
-                    // * Verify the accessibility support in search input
-                    cy.get('#searchChannelsTextbox').should('have.attr', 'placeholder', 'Search channels');
-
-                    // # Focus on the Create Channel button and TAB twice
-                    cy.get('#createNewChannel').focus().tab().tab();
-
-                    // * Verify channel name is highlighted and reader reads the channel name and channel description
-                    cy.get('#moreChannelsList').children().eq(0).as('selectedRow');
-                    cy.get('@selectedRow').within(() => {
-                        cy.get('.more-modal__description').invoke('text').then((description) => {
-                            cy.get('.more-modal__details button').
-                                should('have.class', 'a11y--active a11y--focused').invoke('text').then((channel) => {
-                                    selectedRowText = channel.toLowerCase() + ', ' + description.toLowerCase();
-                                    cy.get('.more-modal__details button').should('have.attr', 'aria-label', selectedRowText);
-                                });
+                cy.get('#moreChannelsModal').
+                    should('be.visible').
+                    and('have.attr', 'role', 'dialog').
+                    and('have.attr', 'aria-labelledby', 'moreChannelsModalLabel').
+                    within(() => {
+                        cy.get('#moreChannelsModalLabel').should('be.visible').and('contain', 'More Channels');
+                        cy.get('.modal-header button.close').should('have.attr', 'aria-label', 'Close');
+
+                        // * Verify the accessibility support in search input
+                        cy.get('#searchChannelsTextbox').should('have.attr', 'placeholder', 'Search channels');
+
+                        cy.waitUntil(() => cy.get('#moreChannelsList').then((el) => {
+                            return el[0].children.length === 2;
+                        }));
+
+                        // # Focus on the Create Channel button and TAB twice
+                        cy.get('#createNewChannel').focus().tab().tab();
```

Comment:
Main fix is here. Before doing tabs, ensure that the list is already loaded with channels.